### PR TITLE
Campaign/applytheme sidebar text active border

### PR DESCRIPTION
**Pull request by deanwhillier**
#### Summary
This PR merges the campaign branch for these completed PRs for a batched QA review:

- https://github.com/mattermost/mattermost-webapp/pull/6712
- https://github.com/mattermost/mattermost-webapp/pull/6699

Developer reviews have already been completed in detail, so this should just be a rubber stamp.

#### Ticket Link

- fixes https://github.com/mattermost/mattermost-server/issues/15864
- fixes https://github.com/mattermost/mattermost-server/issues/15865
- fixes https://github.com/mattermost/mattermost-server/issues/15866

### CODE REVIEW

File: sass/layout/_team-sidebar.scss [hunk 1]
```diff
@@ -187,6 +187,15 @@
...
             .team-container {
+                &::before,
+                &.active::before,
+                &.unread::before {
+                    background: var(--sidebar-text-active-border);
+                }
+            }
+
+            .team-container::before {
+                background: var(--sidebar-text-active-border);
```
Comment (deanwhillier) [inline sass/layout/_team-sidebar.scss:198]: Clean up duplicate CSS.

```suggestion
```

**Comment by deanwhillier** [comment]
FYI @jgilliam17, this PR affects the team icon's blue bar and blue dot:

image

**Review by nevyangelova — APPROVED** [review]

**Review by hmhealey — APPROVED** [review]

**Review by jgilliam17 — APPROVED** [review]
Tested, looks good to merge.
- Verified team sidebar icon bar and dot - as expected, tested in both dark and light themes

**Pull request merged**